package.json

```diff
@@ -109,7 +109,7 @@
     "workbox-webpack-plugin": "^6.4.1"
   },
   "scripts": {
-    "start": "node scripts/start.js",
+    "start": "HTTPS=true node scripts/start.js",
```

Comment:
If the protocol for the host is changed from HTTP to HTTPS, and local storage is keyed on the URL, does this mean that users will lose access to their stored projects that were created prior to this change?

Reply:
"start" script is used to launch the application locally (localhost). "build" script is used in deploying the app in test / prod servers.  I'll revert this change.